app/assets/stylesheets/prism.scss

```diff
@@ -0,0 +1,159 @@
+
```

Comment:
Discussed this in person and the ruby gem doesn't include this.  We've decided that at least a link to the elements repo would be useful and maybe it's worth raising as something to be included in the gem if it's being used for things other than documentation.